# Don't use "-march=native" since the CPU on our build machines may be newer
# than the ones our users have.
--- setup.py
+++ setup.py
@@ -761,7 +761,7 @@ if 'BLOSC' not in optional_libs:
         finally:
             os.remove(fd.name)
 
-    try_flags = ["-march=native", "-msse2"]
+    try_flags = ["-msse2"]
     for ff in try_flags:
         if compiler_has_flags(compiler, [ff]):
             print("Setting compiler flag: " + ff)
